CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+## 2.17.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.2.1890**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented support for monitoring gateway subnets in **aviatrix_gateway** through ``enable_monitor_gateway_subnets`` and ``monitor_exclude_list``
+2. Implemented support for managing Aviatrix VPN timeout configurations through ``idle_timeout`` and ``renegotiation_interval`` in **aviatrix_gateway**
+3. Implemented support for ``enable_active_standby`` in **aviatrix_transit_gateway**
+4. Implemented Active-Standby support for Transit Network workflows:   
+  - ``enable_active_standby`` in **aviatrix_transit_gateway**
+  - ``switch_to_ha_standby_gateway`` in **aviatrix_transit_external_device_conn**
+5. Implemented new resource to decouple ``firewall_instance_association`` out of ``aviatrix_firenet``:
+  - **aviatrix_firewall_instance_association**
+6. Implemented support for transit gateway peering over private networks through the ``enable_peering_over_private_network`` attribute in **aviatrix_transit_gateway_peering**
+7. Implemented support for FQDN gateway in Azure FireNet:
+  - ``fqdn_lan_cidr`` as an attribute, and ``fqdn_lan_interface`` as a computed output in **aviatrix_gateway**
+  - ``lan_interface_cidr`` as an attribute in **aviatrix_transit_gateway**
+8. Implemented support for ``local_login`` in **aviatrix_rbac_group**
+9. Implemented Support for IDP Metadata URLs for SAML endpoints
+10. Implemented support for ``sign_authn_requests`` in **aviatrix_saml_endpoint**
+11. Implemented Bootstrap support for AWS and Azure FireNet solutions in aviatrix_firewall_instance:
+  - ``bootstrap_storage_name``
+  - ``storage_access_key``  
+  - ``file_share_folder``
+  - ``share_directory``
+  - ``sic_key``
+  - ``user_data``
+  - ``container_folder``
+  - ``sas_url_config``
+  - ``sas_url_license``
+12. Implemented support for DH Group 19 in **aviatrix_site2cloud**
+13. Implemented support for Custom Mapped in **aviatrix_site2cloud**
+
+### Enhancements:
+1. Changed ``management_subnet`` to optional to support Check Point and Fortinet instances in **aviatrix_firewall_instance**
+2. Added migration support for following resources due to functionality decoupling:
```

Comment:
What is "migration support" and "functionality decoupling"?

Reply:
"functionality decoupling" is like decouple attaching spoke to transit out of aviatrix_spoke_gateway;
"migration support" is the migration function to write "manage_transit_gateway_attachment" into state file.

Reply:
"Added support for Terraform state migration due to resource-decoupling implementation for the following resources: "

Reply:
oh ok